content/developer/howtos/localization.rst

```diff
@@ -0,0 +1,391 @@
+  `addons/l10n_at/data/account_account_template.xml <{GITHUB_PATH}/addons/l10n_at/data/account_account_template.xml>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_at/data/account_account_template.xml
+    :condition: odoo_dir_in_path
+    :language: xml
+    :start-at: chart_at_template_0010
+    :end-at: </record>
+
+.. warning::
+
+    Avoid the usage of liquidity ``account.account.type``!
+    Indeed, the bank & cash accounts are created directly at the installation of the localization module and then, are linked to an ``account.journal``.
+
+.. warning::
+
+    Only one account of type payable/receivable is enough for the generic case.  We need to define a PoS receivable account as well however. (linked in the CoA)
+
+.. warning::
+
+    Don't create too many accounts: 200-300 is enough.  But mostly, we try to find a good balance where the CoA needs minimal adapting for most companies afterwards.  
+
+Next settings for the chart of accounts are set in a separate file, because we need to provide `list of accounts <#accounts>`__ first. In `data/account_chart_post_data.xml` we set some default accounts:
+
+.. todo add reference to account_id in CoA
+
+.. example::
+  `addons/l10n_ch/data/l10n_ch_chart_post_data.xml <{GITHUB_PATH}/addons/l10n_ch/data/l10n_ch_chart_post_data.xml>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_ch/data/l10n_ch_chart_post_data.xml
+    :condition: odoo_dir_in_path
+    :language: xml
+    :start-at: l10nch_chart_template
+    :end-at: </record>
+
+
+Account groups
+--------------
+
+.. seealso::
+   :ref:`Account Group References <reference/account_group>`
+
+Account groups allow describing the hierarchical structure of the chart of accounts. The filter needs to be activated in the report and then when you decollapse into journal entries it will show the parents of the account.
+
+It works with a prefix start/end, so every account where the code starts with something between start and end will have this account.group as parent group.  Furthermore, the account groups can have a parent account group as well to form the hierarchy.  
+
+
+.. example::
+  `addons/l10n_il/data/account.group.template.csv <{GITHUB_PATH}/addons/l10n_il/data/account.group.template.csv>`__.
+
+  .. csv-table::
+     :condition: odoo_dir_in_path
+     :file: {ODOO_ABSPATH}/addons/l10n_il/data/account.group.template.csv
+     :widths: 20,20,20,20,20
+     :header-rows: 1
+
+Taxes
+-----
+
+.. seealso::
+   - :ref:`Tax References <reference/account_tax>`
+   - :doc:`/applications/finance/accounting/taxation/taxes/taxes`
+
+To add taxes you first need to specify tax groups. You normally need just one tax group for every tax rate, except for the 0% as you need to often distinguish between exempt, 0%, not subject, ... taxes. 
+This model only has two required fields: name and country. Create file `data/account_tax_group_data.xml` and list the groups:
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="utf-8"?>
```

Comment:
Leave out, it's unnecessary syntactic overhead (applies for every literal XML document)